pkg/charts/linkerd2/values_test.go

```diff
@@ -24,6 +24,7 @@ func TestNewValues(t *testing.T) {
 		PrometheusExtraArgs:           map[string]string{},
 		PrometheusAlertmanagers:       []interface{}{},
 		PrometheusRuleConfigMapMounts: []PrometheusRuleConfigMapMount{},
+		PrometheusPersistence:         Persistence{},
```

Comment:
I think you want to populate this with the default values (e.g. ReadWriteOnce and 8Gi).

Reply:
ah I see, thanks!